src/view/pages/Exports/components/ProductCard/index.tsx

```diff
@@ -23,6 +23,13 @@ const PRODUCT_CARD_HEADERS = ['Date/Time', 'Licence Plate #', 'Volume (L)'];
 * Container for Product Card
 */
 export default class ProductCard extends React.Component<PropsType, OwnStateType> {
+
+  private onEntryPress = (uuid: string) => {
+    return () => {
+      // this.props.setActiveMilkEntry(uuid);
+      // this.props.navigateToMilkEntry();
```

Comment:
Why are these commented out?

Reply:
these arn't valid actions for this context I just wanted to leave the example so we know how to implement this in the future